cmd/machine-api-migration/main.go

```diff
@@ -153,6 +166,32 @@ func main() {
 		os.Exit(0)
 	}
 
+	k8sclient := mgr.GetClient()
+	infra := &configv1.Infrastructure{}
+
+	err = k8sclient.Get(stop, client.ObjectKey{Name: controllers.InfrastructureResourceName}, infra)
+	if err != nil {
+		klog.Errorf("failed to fetch infrastructure: %s", err)
+		os.Exit(1)
+	}
+
+	provider, err := getProviderFromInfrastructure(infra)
+	if err != nil {
+		klog.Errorf("failed to fetch infrastructure: %s", err)
+		os.Exit(1)
+	}
+
+	// Currently we only plan to support AWS, so all others are a noop until they're implemented.
+	switch provider {
+	case configv1.AWSPlatformType:
+		klog.Info("MachineAPIMigration: starting AWS controllers")
+
```

Comment:
I think the linter is complaining about this blank line

Reply:
🤦 I forgot to push the linter fixes before re=requesting a review - my bad!